src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -794,6 +794,10 @@ def load_arguments(self, _):
         c.argument('circuit_name', id_part=None)
         c.argument('peering_name', id_part=None)
 
+    with self.argument_context('network express-route peering connection config') as c:
+        c.argument('address_prefix_type', help='The type of address_prefix_type.', arg_type=get_enum_type(['IPv6']))
+        c.argument('address_prefix', help='/125 IP address space to carve out customer addresses for global reach.')
```

Comment:
You can use `network express-route peering connection ipv6-config` and remove `address_prefix_type` argument